src/components/alert/alert.stories.mdx

```diff
@@ -0,0 +1,82 @@
+import { useState } from 'react';
+import { Meta, Story, Preview, Props } from '@storybook/addon-docs/blocks';
+import LinkTo from '@storybook/addon-links/react';
+
+import Alert from '.';
+import Modal from '../modal';
+import Dialog from '../dialog';
+import Button from '../button';
+
+<Meta
+  title="Alert"
+  parameters={{ info: { disable: true }}}
+/>
+
+# Alert
+An important message that interrupts user activity, but can be dismissed.
+
+Useful if a message is so important you’d like to prevent any other activity until the user resolves it.
+
+If the message isn’t so important, or you want to avoid disrupting the user’s activity, consider showing a static Message component in the underlying screen.
+
+Include a Message component within the Alert component, use plain text, or apply a standard Carbon Error or Warning icon.
+
+This component has the same options and properties as the Dialog component.
+
+## Related Components
+- Simple positive or negative confirmation? [Try Flash](/?path=/story/flash--default "Try Flash").
+- Longer message which stays on-screen? [Try Message](/?path=/docs/message--default-story "Try Message").
+- Longer, time sensitive message that must be dismissed? [Try Toast](/?path=/docs/design-system-toast--default-story "Try Toast").
+- Confirm or cancel an action I’ve initiated? [Try Confirm](/?path=/docs/confirm--default-story "Try Confirm").
+- Simple task in context? [Try Dialog](/?path=/docs/dialog--default-story "Try Dialog").
+
+## Quick Start
+
+Import the component and use as shown below:
+
+```jsx
+import Alert from "carbon-react/lib/components/alert"
```

Comment:
Same as Sam mentioned before. It's probably best to put a semi-colon at the end of the line so the consumer can just copy and paste the import.